ckan/lib/helpers.py

```diff
@@ -1010,6 +1007,14 @@ def _range(self, regexp_match):
         return re.sub(current_page_span, current_page_link, html)
 
 
+def get_display_timezone():
+    timezone_name = config.get('ckan.display_timezone', '')
+    if timezone_name == 'server':
+        timezone_name = tzlocal.get_localzone().zone
+
+    return timezone_name
+
+
 def render_datetime(datetime_, date_format=None, with_hours=False):
     '''Render a datetime object or timestamp string as a localised date or
     in the requested format.
```

Comment:
The UnknownTimeZoneError handling below should be dead code now, please remove it too